docs/sources/datasources/grafana-pyroscope.md

```diff
@@ -44,53 +42,71 @@ To configure basic settings for the data source, complete the following steps:
    | `Basic Auth`   | Enable basic authentication to the data source.                                                                                                                                                                                                                                                                          |
    | `User`         | User name for basic authentication.                                                                                                                                                                                                                                                                                      |
    | `Password`     | Password for basic authentication.                                                                                                                                                                                                                                                                                       |
-   | `Minimal step` | Used for queries returning timeseries data. Phlare backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval use Minimal step same or higher than your Phlare scrape interval. For Pyroscope backend this prevents returning too many data points to the front end. |
+   | `Minimal step` | Used for queries returning timeseries data. Phlare backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval, use Minimal step same or higher than your Phlare scrape interval. For Pyroscope backend, this prevents returning too many data points to the frontend. |
```

Comment:
@Rperry2174 Do we need to do anything with the two sentences that refer to Phlare on line 45? Should they be updated in ay way or removed?

Reply:
yeah we should replace phlare with pyroscope everywhere here (and remove any remaining references to phlare

Reply:
I searched through the Pyroscope page and didn't find any more references to Phlare.